types/react-native-dialog/react-native-dialog-tests.tsx

```diff
@@ -17,30 +17,19 @@ class Example extends Component {
     }
 
     singleButton() {
-        return (<Dialog.Button label="Click" onPress={() => null} />);
+        return <Dialog.Button label="Click" onPress={() => null} />;
     }
 
     render() {
         const ref = createRef();
 
         return (
-            <Dialog.Container
-                visible={true}
-                style={{ marginLeft: 20, marginRight: 20 }}
-            >
-                <Dialog.Title style={{ marginLeft: 20, marginRight: 20 }}>
-                    A nice title
-                </Dialog.Title>
+            <Dialog.Container visible={true} style={{ marginLeft: 20, marginRight: 20 }} avoidKeyboard={true}>
```

Comment:
Maybe add some new props from `react-native-modal`. Otherwise test will not reflect these changes